repositories.yaml

```diff
@@ -69,6 +74,7 @@ firefox-for-fire-tv:
     - org.mozilla.components:service-glean
 lib-crash:
   app_id: lib-crash
+  description: A collection of Android libraries to build browsers or browser-like applications
```

Comment:
It would be nice to give these types of component repositories sensible names.

Reply:
Each Android component has its own README and tagline, for example for this one: https://github.com/mozilla-mobile/android-components/tree/master/components/lib/crash

Reply:
Awesome! Integrated those.